distribution/packages/build.gradle

```diff
@@ -333,12 +323,6 @@ Closure commonRpmConfig(boolean oss) {
 
     // without this the rpm will have parent dirs of any files we copy in, eg /etc/elasticsearch
     addParentDirs false
-
-    // Declare the folders so that the RPM package manager removes
-    // them when upgrading or removing the package
-    directory('/usr/share/elasticsearch/bin', 0755)
-    directory('/usr/share/elasticsearch/lib', 0755)
-    directory('/usr/share/elasticsearch/modules', 0755)
```

Comment:
These are now covered in the common package config as it traverses all the way down to `/usr/share/elasticsearch/<foo>`, and in that copyspec `<foo>` is either bin, lib, or modules